fsys_test.go

```diff
@@ -0,0 +1,135 @@
+package skeletonkit_test
+
+import (
+	"embed"
+	"flag"
+	"io"
+	"path/filepath"
+	"strings"
+	"testing"
+	"text/template"
+
+	"github.com/gostaticanalysis/skeletonkit"
+	"github.com/tenntenn/golden"
+)
+
+type appInfo struct {
+	Name       string
+	ModulePath string
+}
+
+type errMap struct {
+	errParse  bool
+	errExec   bool
+	errCreate bool
+}
+
+//go:embed testdata/_template
```

Comment:
Prefix `_` does not need because go command ignores under `testdata` in build time.
If we put directly a template directory under Go Module managed directory, we should put `_` prefix to directory name.